jenkins-jobs/git/git-notify-ib-updates

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+
+import datetime
+import json
+import logging
+import sys
+import os
+import urllib.error
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "shift"))
+try:
+    from github_utils import github_api
+    import libib
+
+    # noinspection PyUnresolvedReferences
+    from libib import PackageInfo, ErrorInfo
+except ImportError:
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "shift"))
+    from github_utils import github_api
+    import libib  # NOQA
+
+    # noinspection PyUnresolvedReferences
+    from libib import PackageInfo, ErrorInfo
+
+try:
+    current_shifter = libib.fetch("/SDT/shifter.txt", content_type=libib.ContentType.TEXT)
+    exit_code = 0
+except (urllib.error.URLError, urllib.error.HTTPError) as e:
+    print("WARNING: failed to get current_shifter!")
+    print(e)
+    current_shifter = "all"
+    exit_code = 1
+
+cache_root = "/build/builds/cms-ib-notifier"
```

Comment:
why hard coded path here? I guess it is path on jenkins controller node ... right? If yes then I guess you can use env `JENKINS_HOME` ( so the full path will be `${JENKINS_HOME}/builds/cms-ib-notifier` ) . Looks like this is a cache directory which is persistant on controller, in tis case I would suggest to use `${JENKINS_HOME}/workspace/cache/cms-ib-notifier` .

Reply:
This path was suggested by Andrea. I will change it following your recommendations.